etc/profile-m-z/mpv.profile

```diff
@@ -79,7 +79,7 @@ seccomp
 seccomp.block-secondary
 tracelog
 
-private-bin env,mpv,python*,waf,youtube-dl,yt-dlp
+private-bin env,lua*,mpv,python*,waf,youtube-dl,yt-dlp
```

Comment:
> Although mpv.profile includes `allow-lua.inc`, lua support was effectively
> broken due to not being referenced in `private-bin`. This PR fixes this.

That may very well be the case, but note that mpv links to libluajit:

```console
$ LC_ALL=C pacman -Sii community/mpv | grep '^Depends On' | tr ' ' '\n' |
  grep lua
luajit
$ pacman -Q mpv
mpv 1:0.35.1-2
$ strings /usr/bin/mpv | grep luajit
libluajit-5.1.so.2
alsa av-channel-layout build-date cdda cplayer cplugins cuda-hwaccel cuda-interop dmabuf-interop-gl dmabuf-interop-pl dmabuf-wayland drm drm-is-kms dvbin dvdnav egl egl-drm egl-helpers egl-x11 ffmpeg ffnvcodec gbm gl gl-wayland glibc-thread-name glob glob-posix gpl iconv jack javascript jpeg jpegxl lcms2 libarchive libass libavdevice libbluray libdl libm libmpv-shared libplacebo
libplacebo-next librt linux-fstatfs luajit manpage-build memfd_create noexecstack pipewire plain-gl posix pulse rubberband rubberband-3 shaderc stdatomic threads uchardet vaapi vaapi-drm vaapi-egl vaapi-libplacebo vaapi-wayland vaapi-x-egl vaapi-x11 vdpau vector vt.h vulkan wayland wayland_protocols_1_24 x11 xv zimg zlib
```

Are you sure that it needs to call a system-wide interpreter?

Did you notice any breakage?

Lua is intended to be small enough to be embedded into compiled programs so as
to not need a separate interpreter (though in distros libraries are usually
dynamically linked anyway if possible).

Reply:
> Did you notice any breakage?

I did get broken functionality with [uosc](https://github.com/tomasklaen/uosc). But it turns out this has nothing to do with firejail. See [this comment](https://aur.archlinux.org/packages/mpv-uosc-git#comment-891533) on the AUR.

Thanks for explaining! Removed lua* again from private-bin and added a comment instead to avoid confusion.